contestantA/src/main/java/edu/asu/diging/contestantA/services/Play.java

```diff
@@ -0,0 +1,123 @@
+package edu.asu.diging.contestantA.services;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+
+@Service
+public class Play {
```

Comment:
the fact that you called the variable of this class in the listener class ```game``` indicates that this is not the best class name.